src/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
-add_library(ReadSpeed INTERFACE)
+add_library(ReadSpeed SHARED
+    ${CMAKE_CURRENT_SOURCE_DIR}/ReadSpeed.cxx
+    ${CMAKE_CURRENT_SOURCE_DIR}/ReadSpeedInput.cxx
```

Comment:
this does not compile now

Reply:
Sorry I made changes and then undid them and forgot to redo them, it's been fixed